.changeset/breezy-carpets-smile.md

```diff
@@ -0,0 +1,5 @@
+---
+'@emotion/jest': major
+---
+
+`test` & `print` are no longer exported as named exports. If you want to access the default serializer just access the default export.
```

Comment:
`test` & `print` were exported so you could add `@emotion/jest` to `snapshotSerializers`, I think I'd like to keep `@emotion/jest` working and throw an error when `@emotion/jest/serializer` is imported that tells people to use `@emotion/jest`

Reply:
Hm, isn't it packing too much into a single module? Having a separate entry point for this seems conceptually easier as with `test` & `print` being exported we kinda trick Jest to think that it's a serializer - structurally ok-ish but nominally not so much.

Reply:
Yeah, okay, sure. Let's use the seperate module. Could you make the changeset clearer that using the raw package name in the `snapshotSerializers` option will no longer work and they'll have to use `@emotion/jest/serializer` instead?

Reply:
added this info, could you re-check?